README.md

```diff
@@ -155,6 +155,18 @@ Specify which format to use for reporting test results. Valid options are:
 elm-test --report json
 ```
 
+### --no-color
+
+> **Note:** `--no-color` and `--color` are broken in version 0.19.1-revision5. See [#491](https://github.com/rtfeldman/node-test-runner/pull/491).
```

Comment:
How would you feel about adding a changelog entry and moving this comment there. I worry that if we add this to the readme it will stay there for ages long after we have forgotten about the revision 5 release?

Reply:
Especially if we make a revison6 straight away.

Reply:
I added it here because I thought it would take a long time to get a new release out. I’ll move it.